MAUI/Scheduler/migration.md

```diff
@@ -496,6 +496,281 @@ this.Content = scheduler;
 
 </table> 
 
+### Scheduler Resource 
+
+The following code example explains how to add the scheduler resource in Xamarin SfSchedule and .NET MAUI SfScheduler.
+
+<table>
+<tr>
+<th>Xamarin SfSchedule</th>
+<th>.NET MAUI SfScheduler</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<syncfusion:SfSchedule
+          x:Name="schedule"
+          ScheduleView="WeekView" 
+          ShowResourceView="True">
+          <syncfusion:SfSchedule.ScheduleResources>
+               <syncfusion:ScheduleResource 
+                    Name="Brooklyn" 
+                    Id="5601" 
+                    Color="#FF3399" />
+          </syncfusion:SfSchedule.ScheduleResources>
+</syncfusion:SfSchedule>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+// Creating an instance for schedule resource collection.
+ObservableCollection<object> resources = new ObservableCollection<object>();
+
+// Adding schedule resource in schedule resource collection.
+resources.Add(new ScheduleResource() 
+{ 
+     Name = "Brooklyn", 
+     Id = 5601, 
+     Color = Color.FromHex("#FF3399") 
+});
+
+// Adding schedule resource collection to schedule resources of SfSchedule.
+schedule.ScheduleResources = resources;
+
+// Creating an instance for schedule appointment collection
+ScheduleAppointmentCollection scheduleAppointmentCollection = new ScheduleAppointmentCollection();
+//Adding schedule appointment in schedule appointment collection 
+scheduleAppointmentCollection.Add(new ScheduleAppointment()
+{
+     StartTime = new DateTime(2019, 05, 08, 10, 0, 0),
+     EndTime = new DateTime(2019, 05, 08, 12, 0, 0),
+     Subject = "Meeting",
+     Location = "Hutchison road",
+     ResourceIds = new ObservableCollection<object> { 5601}
+});
+
+//Adding schedule appointment collection to DataSource of SfSchedule
+schedule.DataSource = scheduleAppointmentCollection;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<scheduler:SfScheduler View="TimelineWeek" AppointmentsSource="{Binding Appointments}"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+public ObservableCollection<SchedulerAppointment> Appointments { get; set; }
+
+SfScheduler scheduler = new SfScheduler();
+// Creating an instance for the scheduler appointment collection.
+this.Appointments = new ObservableCollection<SchedulerAppointment>();
+
+// Adding scheduler appointment in the schedule appointment collection. 
+Appointments.Add(new SchedulerAppointment()
+{
+    StartTime = DateTime.Today.AddHours(9),
+    EndTime = DateTime.Today.AddHours(11),
+    Subject = "Client Meeting",
+    Background = Brush.LightSkyBlue,
+});
+
+// Adding the scheduler appointment collection to the AppointmentsSource of .NET MAUI Scheduler.
```

Comment:
//Adding the scheduler appointment collection to the AppointmentsSource of the .NET MAUI Scheduler.